src/identifier.ts

```diff
@@ -1,10 +1,29 @@
 import qs from 'qs';
 import client from './api';
-import Assignment from './assignment';
+import Assignment, { AssignmentData } from './assignment';
 import Visitor from './visitor';
 
+export type IdentifierOptions = {
+  visitorId: string;
+  identifierType: string;
+  value: number;
```

Comment:
value should be a string (https://github.com/Betterment/test_track/blob/master/db/schema.rb#L135). there are a few other places that needed to be updated to match.